src/plugins/saved_objects/public/saved_object/saved_object_loader.ts

```diff
@@ -132,15 +132,16 @@ export class SavedObjectLoader {
     search: string = '',
     size: number = 100,
     fields?: string[],
-    hasReference?: SavedObjectsFindOptions['hasReference']
+    hasReference?: SavedObjectsFindOptions['hasReference'],
+    searchFields: string[] = ['title^3', 'description']
```

Comment:
Can you update the jsdoc comment above to include the new param? This is a nice improvement to this interface!

Reply:
added